two-joint-ik-solver.js

```diff
@@ -106,53 +94,65 @@ export class TwoJointIkSolver extends Component {
 
     time = 0;
 
+    rootScaling = new Float32Array(3);
+    tempQuat = new Float32Array(4);
```

Comment:
This one can stay a global temporary. The others should have indeed been members all along 🤔